src/styles/blocks/page.css

```diff
@@ -0,0 +1,24 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(../../fonts/Avenir.ttc),
+    url(../../fonts/Avenir-Book.ttf),
+    url(../../fonts/Avenir-Heavy.ttf);
+}
+
+.page {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.page__body {
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-top: 20px;
+
+  display: flex;
+  flex-direction: column;
+}
+
+.page__search-bar:nth-of-type(n + 2) {
```

Comment:
if you plan to add elements and want to give them `margin-top: 20px`, it is better to use `:not(:first-child)`